financial-services-accelerator/components/org.wso2.financial.services.accelerator.consent.mgt.dao/src/main/java/org/wso2/financial/services/accelerator/consent/mgt/dao/constants/ConsentMgtDAOConstants.java

```diff
@@ -0,0 +1,157 @@
+    public static final String AUTH_TYPE = "AUTH_TYPE";
+    public static final String USER_ID = "USER_ID";
+    public static final String AUTH_STATUS = "AUTH_STATUS";
+    public static final String AUTH_UPDATED_TIME = "AUTH_UPDATED_TIME";
+    public static final String MAPPING_ID = "MAPPING_ID";
+    public static final String ACCOUNT_ID = "ACCOUNT_ID";
+    public static final String PERMISSION = "PERMISSION";
+    public static final String MAPPING_STATUS = "MAPPING_STATUS";
+    public static final String ATT_KEY = "ATT_KEY";
+    public static final String ATT_VALUE = "ATT_VALUE";
+    public static final String SESSION_DATA_KEY = "sessionDataKey";
+    public static final String CONSENT_FILE = "CONSENT_FILE";
+    public static final String AUTH_MAPPING_ID = "AUTH_MAPPING_ID";
+    public static final String ACTION_BY = "ACTION_BY";
+    public static final String ACTION_TIME = "ACTION_TIME";
+    public static final String STATUS_AUDIT_ID = "STATUS_AUDIT_ID";
+    public static final String PREVIOUS_STATUS = "PREVIOUS_STATUS";
+    public static final String REASON = "REASON";
+    public static final String RECORD_ID = "RECORD_ID";
+    public static final String EFFECTIVE_TIMESTAMP = "EFFECTIVE_TIMESTAMP";
+    public static final String TABLE_ID = "TABLE_ID";
+    public static final String HISTORY_ID = "HISTORY_ID";
+    public static final String CHANGED_VALUES = "CHANGED_VALUES";
+    public static final String CONSENT_EXPIRY_TIME_ATTRIBUTE = "ExpirationDateTime";
+
+    public static final String TYPE_CONSENT_BASIC_DATA = "ConsentData";
+    public static final String TYPE_CONSENT_AUTH_RESOURCE_DATA = "ConsentAuthResourceData";
+    public static final String TYPE_CONSENT_ATTRIBUTES_DATA = "ConsentAttributesData";
+    public static final String TYPE_CONSENT_MAPPING_DATA = "ConsentMappingData";
+    public static final String CONSENT_IDS = "consentIDs";
+    public static final String CLIENT_IDS = "clientIDs";
+    public static final String CONSENT_TYPES = "consentTypes";
+    public static final String CONSENT_STATUSES = "consentStatuses";
+    public static final String USER_IDS = "userIDs";
+    public static final String IN = "inOperator";
+    public static final String AND = "andOperator";
+    public static final String OR = "orOperator";
+    public static final String WHERE = "where";
+    public static final String PLACEHOLDER = "placeholder";
+    public static final String PLAIN_PLACEHOLDER = "plainPlaceholder";
+    public static final String EQUALS = "equals";
+    // Consent Database Table Identifiers
+    public static final String TABLE_OB_CONSENT = "OB_CONSENT";
+    public static final String TABLE_OB_CONSENT_AUTH_RESOURCE = "OB_CONSENT_AUTH_RESOURCE";
+    public static final String TABLE_OB_CONSENT_MAPPING = "OB_CONSENT_MAPPING";
+    public static final String TABLE_OB_CONSENT_ATTRIBUTE = "OB_CONSENT_ATTRIBUTE";
+    public static final String TABLE_OB_CONSENT_FILE = "OB_CONSENT_FILE";
```

Comment:
Are we still using the OB prefixes?

Reply:
FIxed in https://github.com/wso2/financial-services-accelerator/pull/108/commits/74e401a23f66920af64aee5717dd4ccc8a075948